app/src/main/java/org/mozilla/rocket/privately/PrivateTabViewProvider.kt

```diff
@@ -12,6 +13,11 @@ class PrivateTabViewProvider(private val host: Activity) : TabViewProvider {
         return WebViewProvider.create(host, null, WebViewSettingsHook) as TabView
     }
 
+    override fun purify() {
+        create().cleanup()
+        CookieManager.getInstance().removeAllCookies(null)
```

Comment:
Looks like WebkitView.cleanup already includes CookieManager.removeAllCookies. Do we still need to do it here?

Reply:
I was thinking to do that explicitly, but remove that also make sense to me.